vendor/github.com/oracle/bmcs-go-sdk/object_storage_objects.go

```diff
@@ -204,6 +204,15 @@ func (c *Client) PutObject(
 	}
 	required.Body = content
 
+	// application/json gets added by default if content-type is not specified. This is not desirable
```

Comment:
So the service will set "application/octet-stream" if it doesn't get a content type, and we're just mirroring that here, correct?

Reply:
The sdk sets application/json if no content type is set which is correct/required for basically every other end point. For object store that is not a correct default and must have an explicit override of application/octet-stream (if none is supplied by the user). 

Ideally we could tell the sdk to not set a default content-type here but it is not sufficiently parameterized to support that.